bubblewrap.c

```diff
@@ -1327,6 +1333,10 @@ main (int    argc,
   if (!is_privileged)
     opt_unshare_user = TRUE;
 
+  if (opt_unshare_user_try &&
+      stat ("/proc/self/ns/user", &sbuf) == 0)
```

Comment:
Please actually *try* the RHEL7 case.  What I've been saying this whole time is that `--unshare-user-try` *does* break on RHEL7 privileged because even uid 0 can't `CLONE_NEWUSER` unless the module parameter has been set.

With your patches:

```
$ id
uid=1000(walters) gid=1000(walters) groups=1000(walters),10(wheel),135(mock),156(stapusr),157(stapsys),158(stapdev) context=unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023
$ cat /etc/redhat-release 
Red Hat Enterprise Linux Workstation release 7.2 (Maipo)
$ ls -al /usr/bin/bwrap
-rwsr-xr-x. 1 root root 87407 May 26 15:49 /usr/bin/bwrap
$ uname -r
3.10.0-327.13.1.el7.x86_64
$ bwrap --ro-bind / / --dev /dev /bin/true; echo $?
0
$ bwrap --ro-bind / / --dev /dev --unshare-user-try /bin/true; echo $?
Creating new namespace failed, likely because the kernel does not support user namespaces.  bwrap must be installed setuid on such systems.
1
```